tests/test_cbv.py

```diff
@@ -0,0 +1,84 @@
+import unittest
```

Comment:
Since tests are in example folder, could you remove it?

Reply:
The example folder is misleading tests should be in a `tests` folder. But i would still leave the example folder,

Reply:
Ok, let's move the example in test folder.